en/advanced/saveactions.md

```diff
@@ -146,3 +146,14 @@ Changes all letters to upper case.
 
 Shortens lists of persons if there are more than 2 persons to \"et al.\".
 
+## Remove digits
+
+Removes all digits from the field content.
```

Comment:
Could you please add examples here?

IMHO you can even embed BibTeX in markdown.

    ```bibtex
    @article{...,
    }
    ```

Having full examples would really help the reader.

In case you don't have the time to make up full examples, it would be helpful to include one example from the Java tests here. (`For instance, "Stefan Kolb 0001" gets Stefan Kolb"`)

Reply:
I think bibtex code would be overkill here (since it operates only on the field content).

Reply:
I am thinking as a user also working with text editors or GitHub pull requests with BibTeX files. I would just put two lines here to provide the idea. Maybe author and title. Not a full BibTeX.

Reply:
@tobiasdiez One night later 😅: OK, just the text change without BibTeX. The overall presentation etc we can do in a f2f discussion!